Accordion: Removes Animation

anechol opened the pull request:
# Description

Removes any code related to animation. Animation for the accordion can be revisited at a later date.


## Type of change

Please delete options that are not relevant.
If your type of change is not present, add that option.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests you've added and run to verify your changes.
Provide instructions so that we can reproduce.
Please also list any relevant details for your test configuration.

- [ ] unit tests
- [ ] e2e tests
- [ ] accessibility tests
- [x] tested manually
- [ ] other:

**Test Configuration**:

- Pine versions:
- OS:
- Browsers:
- Screen readers:
- Misc:

# Checklist:

If not applicable, leave options unchecked.

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes
- [ ] Design has QA'ed and approved this PR

monkeypox8 commented on app/assets/stylesheets/sage/system/core/_mixins.scss:
```diff
@@ -152,7 +152,7 @@
   display: inline-flex;
   text-transform: none;
-  font: normal normal normal 1rem/1 "Sage";
+  font: normal normal normal 1em/1 "Sage";
```
This change could mean we have to reconsider scenarios with unwanted icon scaling, which were prevented through using a fixed size.

Commit:
fix(accordion): removes accordion animation

Commit:
style(accordion): removes unneeded styles

ju-Skinner commented on libs/core/src/components/pds-accordion/pds-accordion.tsx:
```diff
@@ -24,33 +23,23 @@ export class PdsAccordion {
-    mutable: true,
-    reflect: true
-  }) isOpen: boolean = false;
+  @Prop({ attribute: 'open', mutable: true, reflect: true }) isOpen: boolean = false;
```
Minor suggestion, but legibility was better the other way around when on multiple lines.

anechol replied:
@ju-Skinner I agree, I think I need to adjust some linting settings in VS Code because it was complaining about it not being on one line.

QuintonJason reviewed (approved):
Good to go

kajabi-bot committed:
Publish

 - @pine-ds/core@1.1.0-rc.0
 - @pine-ds/doc-components@1.1.0-rc.0
 - @pine-ds/react@1.1.0-rc.0

Commit:
chore: fixes prop formatting

Commit:
chore: fixes linting issue
